### PR TITLE
Add a helper function to get dtype

### PR DESCRIPTION
**Pull request by jeremykubica**
Add a simple helper function to `parse_arguments.py` to extract the dtype (and do basic error checking). Replaces the conditionals throughout the test scripts.

### CODE REVIEW

File: tests/deepdisc/utils/test_parse_arguments.py [hunk 1]
```diff
@@ -50,3 +55,13 @@ def test_make_training_arg_parser():
...
+
+
+def test_dtype_from_args():
+    assert dtype_from_args(8) is np.uint8
+    assert dtype_from_args(16) is np.int16
+
+    with pytest.raises(ValueError):
+        _ = dtype_from_args(4)
+    with pytest.raises(ValueError):
+        _ = dtype_from_args(32)  # Do we want to support this too?
```
Comment (delucchi-cmu) [inline tests/deepdisc/utils/test_parse_arguments.py:67]: Or any kind of floating points?

Comment (grantmerz) [reply tests/deepdisc/utils/test_parse_arguments.py:67]: We should support float32 data.  Currently 'raw' data scaling will return a float32 array without regarding the dtype argument.

Comment (jeremykubica) [reply tests/deepdisc/utils/test_parse_arguments.py:67]: What is the best way to do that? I could map an input of 32 to that type. But will that cause possible confusion with int32s? Alternatively I could have float32 be a default and not raise an error if no type is given.

Any thoughts?

**Review by delucchi-cmu — APPROVED** [review]

File: tests/deepdisc/utils/test_parse_arguments.py (hunk 1 above)
Comment (delucchi-cmu) [reply tests/deepdisc/utils/test_parse_arguments.py:67]: or take in a string like "int32"/"float32"?

Comment (grantmerz) [reply tests/deepdisc/utils/test_parse_arguments.py:67]: I think defaulting to float32 is ok, it's likely we won't use int32

Comment (jeremykubica) [reply tests/deepdisc/utils/test_parse_arguments.py]: I made the default float32 and updated the tests.

**Pull request merged**